src/main/java/com/aws/greengrass/clientdevices/auth/metrics/ClientDeviceAuthMetrics.java

```diff
@@ -5,70 +5,62 @@
 
 package com.aws.greengrass.clientdevices.auth.metrics;
 
-
 import com.aws.greengrass.telemetry.impl.Metric;
+import com.aws.greengrass.telemetry.impl.MetricFactory;
 import com.aws.greengrass.telemetry.models.TelemetryAggregation;
 import com.aws.greengrass.telemetry.models.TelemetryUnit;
 
+import java.time.Clock;
 import java.time.Instant;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicLong;
+import javax.inject.Inject;
 
 public class ClientDeviceAuthMetrics {
+    private final AtomicLong subscribeToCertificateUpdatesSuccess = new AtomicLong();
+    private final MetricFactory mf = new MetricFactory(NAMESPACE);
+    private final Clock clock;
+    private static final String NAMESPACE = "ClientDeviceAuth";
```

Comment:
`ClientDevicesAuth` (missing "s"). Is this the namespace we're using or should it be`aws.greengrass.clientdevices.Auth` ?